rm-datasource/src/main/java/io/seata/rm/datasource/undo/oracle/keyword/OracleKeywordChecker.java

```diff
@@ -28,29 +29,12 @@
  * @date 2019/3/25 oracle keyword checker
  */
 public class OracleKeywordChecker implements KeywordChecker {
-    private static volatile KeywordChecker keywordChecker = null;
-    private Set<String> keywordSet;
+    private static Set<String> keywordSet;
 
-    private OracleKeywordChecker() {
+    static {
```

Comment:
why is constructor update to static

Reply:
In old way. the keywordSet will be initialized when the private ctor executed and will be initialized only once(by the lock). 
Now the constructor is set to public, so the initialization of the keywordset is put into the static block to ensure that it is executed only once.